source/dev_guide/testing.rst

```diff
@@ -0,0 +1,73 @@
+Testing E3SM Diagnostics
+==============================================
+
+Unit tests
+----------
+
+Run all unit tests by doing the following:
+
+    .. code::
+
+        pip install . # Install your changes
+        ./tests/test.sh # Run all unit tests
+
+Automated tests
+---------------
+
+We have a Continuous Integration / Continuous Delivery (CI/CD) workflow.
+The unit tests are run automatically as part of this.
+
+Complete run test
+-----------------
+
+``tests/complete_run_v2_3_0.py`` checks the images generated by all diagnostics to
+see if any differ from expected.
+This test is not run as part of the unit test suite, because it relies on a large
+quantity of data found on LCRC (Anvil/Chrysalis).
+
+    .. warning::
+        You have to run this test manually. It is not run as part of the CI/CD workflow.
+
+If you've been developing code on a different machine, you can get the code to a LCRC
+machine by doing the following. First, push your code to GitHub.
+Then log into a LCRC machine and do the following:
+
+    .. code::
+
+        git fetch <repo-name> <branch-name> # Fetch the branch you just pushed
```

Comment:
`<fork-name>` might be better than `<repo-name>` here.

Reply:
I agree since we use the forking model and should run the tests against the fork